apps/crn-server/test/integration/fixtures/contentful.ts

```diff
@@ -389,12 +389,12 @@ export class ContentfulFixture implements Fixture {
       content_type: 'events',
     });
 
-    events.forEach(async (entry) => {
+    for (const entry of events) {
```

Comment:
if this is a fix not part of the switch then maybe best to take it out in case we were revert this commit/PR at some point or whatever